index.html

```diff
@@ -94,6 +94,36 @@ <h2>
         the end-user's preferences.
       </p>
     </section>
+    <section>
+      <h2>
+        Infrastructure
+      </h2>
+      <section>
```

Comment:
Put `data-dfn-for="processed image resource"` on this `section` tag, then you don't need it in the individual `dfn`s below.

Reply:
I couldn't get rid of a dfn collision error with the ECMASCRIPT Type() dfn, which is why I had to inline them. It works if they are inlined for some reason.